compiler/src/dotty/tools/dotc/core/quoted/PickledQuotes.scala

```diff
@@ -122,7 +195,13 @@ object PickledQuotes {
     unpickler.enter(Set.empty)
 
     val tree = unpickler.tree
-    quotePickling.println(i"**** unpickle quote ${tree.show}")
+
+    // Make sure trees and positions are fully loaded
+    new TreeTraverser {
+      def traverse(tree: Tree)(implicit ctx: Context): Unit = traverseChildren(tree)
+    }.traverse(tree)
```

Comment:
What will happen if we remove the code above? Maybe add a doc for consequences if the action is not taken.

Reply:
Some symbols and positions are not loaded early enough. Not sure why though. But it has been like that since the start. Dimitry mentioned the need for this for loading trees from tasty in general and we have keep this mechanism so far.